src/Codeception/Lib/Connector/Guzzle6.php

```diff
@@ -338,4 +349,16 @@ public static function createHandler($handler)
         }
         return HandlerStack::create();
     }
+
+    public function setAwsAuth($config)
+    {
+        try {
+            $this->awsCredentials = new Credentials($config['key'], $config['secret']);
+            $this->awsSignature = new SignatureV4($config['service'], $config['region']);
+        } catch (CredentialsException $e) {
+            throw $e;
+        } catch (UnresolvedSignatureException $e) {
```

Comment:
Why do you catch these 2 exception if you do nothing with them?

Reply:
so the idea was I wanted to catch if a key was not set. This was not the right location to do it. And yes just catching and throwing does not help.